doc/test/dependencies.tex

```diff
@@ -1,22 +1,19 @@
-\documentclass{report}
-
-\usepackage{amssymb,amsmath}
-\usepackage[math]{mathspec}
-\usepackage[margin=0.75in, top=1in]{geometry}
-\usepackage{graphicx,grffile}
-\usepackage[table,svgnames]{xcolor}
-\usepackage{framed}
-\usepackage{longtable,booktabs}
-\usepackage{multirow}
-\usepackage{listings}
-\usepackage{enumitem}
-\usepackage[per-mode=symbol]{siunitx}
-\PassOptionsToPackage{hyphens}{url}\usepackage[breaklinks=true]{hyperref}
+\input{../header}
```

Comment:
That makes more sense doesn't it? That way there's no risk of falling out of sync with our actual packages